hooks/blueprint

```diff
@@ -96,11 +96,24 @@ database=0
 for want in ${GENESIS_REQUESTED_FEATURES}; do
   case "$want" in
   # Database selector feature flags
-  mysql-db|postgres-db|local-db)
+  mysql-db|postgres-db)
     database=$(( database + 1 ))
     merge+=( manifests/db/${want%-db}.yml )
     ;;
 
+  local-db|local-ha-db)
+    database=$(( database + 1 ))
+    merge+=( manifests/db/local.yml )
+    case "$want" in
+      local-db)
+        merge+=( manifests/db/local-single.yml )
+        ;;
+      local-ha-db)
+        merge+=( manifests/db/local-ha.yml )
+        ;;
+    esac
+    ;;
+
```

Comment:
Minor tweak: split this into two case clauses, one for `local-db` and one for `local-ha-db`.

`local-db` is local.yml + local-single.yml (in merge)

`local-ha-db` is local.yml + local-ha.yml (in merge)

If the operator activates both, it should be a failure, because we can't have both an internal DB _AND_ an internal HA DB.